### PR TITLE
add json validation

### PR DESCRIPTION
**Pull request by K-Meech**
fixes https://github.com/mobie/mobie-io/issues/70

Adds validation of the ome-zarr '.zattrs' file to the tests.
@constantinpape - there didn't seem to be any Java libraries that support the very latest json schema (2020-12), so here I validate against Draft 7. I tested it with some small modifications + all seemed to work correctly though.

### CODE REVIEW

**Comment by constantinpape** [comment]
> @constantinpape - there didn't seem to be any Java libraries that support the very latest json schema (2020-12), so here I validate against Draft 7.

Draft 7 should be fine, I don't think we need any of the more recent features here.

File: src/test/resources/schema/ome_zarr_0.4.schema [hunk 1]
```diff
@@ -0,0 +1,233 @@
+{
```
Comment (constantinpape) [inline src/test/resources/schema/ome_zarr_0.4.schema:1]: Ideally you wouldn't copy the scheme, but would download it from `"https://ngff.openmicroscopy.org/0.4/schemas/image.schema"` again to pull in potential updates. But for now we can also just copy it to keep this simple.

**Review by constantinpape — APPROVED** [review]

File: src/test/resources/schema/ome_zarr_0.4.schema (hunk 1 above)
Comment (K-Meech) [reply src/test/resources/schema/ome_zarr_0.4.schema:1]: Good point - let me see if there's an easy fix for this.

Comment (K-Meech) [reply src/test/resources/schema/ome_zarr_0.4.schema]: Ok - wasn't too hard to fix this! I download the schema directly from github now

**Comment by constantinpape** [comment]
Thanks @K-Meech. This is good to go then from my side.

**Pull request merged**